inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_f16.cl

```diff
@@ -383,15 +401,38 @@ KERNEL(convolution_bfyx_f16)(
 #if OUTPUT_LEFTOVERS
     if ((feature_block + 1) * FEATURE_SLICE_SIZE >= OUTPUT_FEATURE_NUM) {
         for (int i = 0; i < OUTPUT_X_BLOCK_SIZE; i++) {
+
 #if HAS_FUSED_OPS
             FUSED_OPS_SCALAR;
+#   if OUTPUT_FORMAT_BFYX
+#       if INPUT0_IS_FP && !OUTPUT_IS_FP
+            // Follow reorder_data.cl
+            res[i] = TO_OUTPUT_TYPE(round(FUSED_OPS_RESULT_SCALAR));
```

Comment:
Is this round really needed? I have a feeling that for fsv16 int8 output we don't do the round, so I don't see any reason to do it for bfyx output layout. Actually, reorder is not supposed to do the fp -> int8 conversion as we need quantize primitive to do that.

Reply:
I've created the situation fp-->int8 on unittest, and It returns error and I've added the code to solve it.
If the situation is not realistic as you mentioned, this #if code is not necessary.

Reply:
Removed round()